app/scraping.py

```diff
@@ -1,19 +1,40 @@
 from bs4 import BeautifulSoup
 import os
+import requests
+from PIL import Image
+from io import BytesIO
 
 
 def extract_images(soup: BeautifulSoup) -> list:
     images = []
     n_images = int(os.getenv("N_IMAGES", 4))  # Default to 4 if not set
+    min_width = int(os.getenv("MIN_IMAGE_WIDTH", 0))
```

Comment:
Make this global  variables with 300x250

Reply:
Thank you for the feedback! I'll update the code to use global variables with default values of 300x250 for `MIN_IMAGE_WIDTH` and `MIN_IMAGE_HEIGHT`. I'll make the necessary changes and push the updates shortly.